ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.14.yaml

```diff
@@ -88,25 +94,11 @@ tests:
       masters: "3"
       workers: "2"
     test:
-    - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE415},${RELEASE_IMAGE_INTERMEDIATE416},${RELEASE_IMAGE_TARGET}"
-        > ${SHARED_DIR}/upgrade-edge
-      dependencies:
-      - env: RELEASE_IMAGE_INTERMEDIATE416
-        name: release:intermediate416
-      - env: RELEASE_IMAGE_INTERMEDIATE415
-        name: release:intermediate415
-      - env: RELEASE_IMAGE_TARGET
-        name: release:arm64-target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
+    - ref: cucushift-upgrade-setedge-4hops
```

Comment:
Per [comment](https://github.com/openshift/release/pull/58433#discussion_r1827248316), arm job is excluded currently for `cucushift-upgrade-setedge-4hops`